recipe/rename_psycopg2_binary.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+"""
+Rename package from psycopg2 to psycopg2-binary so pip check will pass for
+package that have psycopg2-binary as a dependency
+"""
+
+with open('setup.py', 'rb') as fo:
+    content = fo.read()
+    content = content.replace(b'name="psycopg2"', b'name="psycopg2-binary"')
+
+with open('setup.py', 'wb') as fo:
+    fo.write(content)
```

Comment:
@bollwyvl, I think this will finally take care of #37 

I'm renaming the package `psycopg2-binary` in `setup.py` (but only for the build of `psycopg2-binary`, not `psycopg2`).  So you will need to make packages that list `psycopg2-binary` as a dependency upstream also depend on that version of the package on conda-forge.